Applications/DataExplorer/VtkVis/VtkRaster.cpp

```diff
@@ -126,6 +120,7 @@ vtkImageAlgorithm* VtkRaster::loadImageFromTIFF(const std::string& fileName,
 
         int version[3];
         int count (0);
+        double x0, double y0, cellsize;
```

Comment:
Should be moved into the `if (geoTiff)` branch.
 ```suggestion
        double x0 = 0.0; 
        double y0 = 0.0;
        double cellsize = 0.0;
```

Reply:
✔️